app/src/main/java/org/standardnotes/notes/comms/Crypt.java

```diff
@@ -223,6 +223,27 @@ public static EncryptedItem encrypt(Note note) {
         return null;
     }
 
```

Comment:
I'm gonna merge this as-is, but it's going to need a bit of refactoring with the tag branch because we will have some duplication